drivers/wifi/nrf700x/zephyr/src/qspi/src/rpu_hw_if.c

```diff
@@ -204,8 +204,12 @@ int rpu_pwron(void)
 		LOG_ERR("IOVDD GPIO set failed...\n");
 		return ret;
 	}
-	/* Settling time for iovdd switch (TCK106AG): ~600us */
-	k_msleep(1);
+	/* Settling time for iovdd DK/EK - switch (TCK106AG): ~600us
```

Comment:
May be prefix with nRF7002 DK/EK and a newline to separate out EB?

Reply:
Done